frontend/lib/norent/data/faqs-content.tsx

```diff
@@ -444,49 +495,55 @@ export const FaqsContent: Faq[] = [
           You should continue to collect documentation that shows you’ve been
           financially impacted by COVID-19.
         </p>
-      </>
+      </Trans>
     ),
   },
   {
-    question: "Will I still owe my rent after I send this letter?",
+    question: li18n._(t`Will I still owe my rent after I send this letter?`),
     category: "After Sending Your Letter",
     answerFull: (
-      <p>
-        Yes, you’ll still owe rent after sending the letter because our state
-        and federal governments have not adopted a rent cancellation policy.
-        Call for cancellation of rent at <CancelRentLink />.
-      </p>
+      <Trans>
+        <p>
+          Yes, you’ll still owe rent after sending the letter because our state
+          and federal governments have not adopted a rent cancellation policy.
+          Call for cancellation of rent at <CancelRentLink />.
+        </p>
+      </Trans>
     ),
   },
   {
-    question: "Help! My landlord is already trying to evict me.",
+    question: li18n._(t`Help! My landlord is already trying to evict me.`),
     category: "After Sending Your Letter",
     answerFull: (
-      <p>
-        If your landlord is trying to illegally evict you, reach out to legal
-        assistance at <LawHelpLink /> immediately.
-      </p>
+      <Trans>
+        <p>
+          If your landlord is trying to illegally evict you, reach out to legal
+          assistance at <LawHelpLink /> immediately.
+        </p>
+      </Trans>
     ),
   },
   {
-    question:
-      "What kind of documentation should I collect to prove I can’t pay rent?",
+    question: li18n._(
+      t`What kind of documentation should I collect to prove I can’t pay rent?`
+    ),
     category: "After Sending Your Letter",
     answerFull: <NonpaymentDocumentation />,
   },
   {
-    question: "What if I live in a state without an eviction moratorium?",
+    question: li18n._(
+      t`What if I live in a state without an eviction moratorium?`
+    ),
     category: "States with Limited Protections",
     answerFull: (
-      <>
+      <Trans>
```

Comment:
For especially long passages of text like the one that starts here, I wonder if it'd be useful to assign an `id` attribute to this [`<Trans>`](https://lingui.js.org/ref/react.html#trans) element.  Normally the id is just the content of the string, which works well for short snippets of text, but big ones like this could potentially really increase the size of our locale bundles, since ultimately they're an object that maps IDs to translations (you can see what it looks like if you run `yarn lingui:compile` and then look at `locales/en/messages.js`)...

For the translation of this piece of content, we'd have an entry in an object that's like `{"...3 paragraphs of text...": "...translation of 3 paragraphs of text..."}` which might be better as e.g. `{"answer.connect-with-someone": "...translation of 3 paragraphs of text..."}`.  Also, because I _think_ CrowdIn shows the translator the id, it might allow them to get additional helpful context from it (e.g. that it's the answer to a FAQ question) that they might not otherwise.  Just a thought tho.